charts/cray-etcd-base/templates/wait-for-job.yaml

```diff
@@ -43,26 +43,12 @@ spec:
           command:
           - /bin/sh
           - -c
-          {{- if .Values.etcd.auth.client.secureTransport }}
-          - until etcdctl --ca-file /tls/ca.crt --cert-file /tls/tls.crt --key-file /tls/tls.key --endpoints https://$ETCD_HOST:$ETCD_PORT member list | grep started | wc -l | grep -q $ETCD_NUM_REPLICAS; do echo "Waiting for etcd cluster"; sleep 2; done; echo 'ETCD READY';
-          {{- else }}
```

Comment:
Is `secureTransport` (`https`) no longer supported?  Or just not needed here?  There are still other references to `secureTransport`.
```
./charts/cray-etcd-base/tests/etcd-enabled_test.yaml:      etcd.auth.client.secureTransport: true
./charts/cray-etcd-base/templates/_helpers.tpl:{{- if .Values.etcd.auth.peer.secureTransport -}}
./charts/cray-etcd-base/templates/_helpers.tpl:{{- if .Values.etcd.auth.client.secureTransport -}}
./charts/cray-etcd-base/values.yaml:      secureTransport: false
./charts/cray-etcd-base/values.yaml:      secureTransport: false
```

Reply:
When I was testing this feature, I found that it did not work. I talked with the teams using the chart and they were not interested in using it. I removed it since people were not interesting in using it and it did not work properly as it was set up. 
What are the other references you to see to `secureTransport`?

Reply:
The 5 lines I mentioned in my comment, I did a grep through the branch for `secureTransport`

Reply:
I see you've removed the rest of the `secureTransport` references, thanks!